apollo/plugin.py

```diff
@@ -188,12 +191,12 @@ def apollo_people_enrichment(
 
     data = {key: value for key, value in data.items() if value is not None}
 
-    with httpx.Client(
-        transport=AugmentedTransport(
+    async with httpx.AsyncClient(
+        transport=AsyncAugmentedTransport(
             actions_v0.authenticated_request_apollo_request_body_auth
         ),
     ) as client:
-        response = client.post(url, json=data, headers=headers)
-        response.raise_for_status()
-        result = response.json()
-        return _parse_people_enrichment_data(result.get("person", {}))
+        response = await client.post(url, json=data, headers=headers)
+        await response.aread()
+        data = response.raise_for_status().json()
```

Comment:
Raise for status before reading?  If you agree, do this consistently throughout this PR.

Reply:
Done, except in cases where we have custom error handlers already (airtable, and hubspot) -- I am not sure about the hubspot handler (our requests library has a error handler that does seems a bit incongruous -- I can fix that separately if we want to change it.)